src/utils/winner.js

```diff
@@ -0,0 +1,15 @@
+export const winner = carsInfo => {
+	let maxLength = 0
+	let winnerArr = []
```

Comment:
```suggestion
	const winnerArr = []
```
`const`로 선언해줘도 좋을 것 같아요!